### PR TITLE
[Enhancement] Refinements to project summary status update

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/12832 -- format long titles
https://github.com/cityofaustin/atd-data-tech/issues/10596 -- style changes to project summary status updates

## Testing
**URL to test:** 

https://deploy-preview-1079--atd-moped-main.netlify.app/moped/projects/1950

**Steps to test:**

Project 1950 has a super long title, check to see that the title doesnt overlap with the project tabs
The status update author's name shouldn't be bolded, and the timestamp should be relative to now. 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js [hunk 1]
```diff
@@ -60,6 +60,12 @@ const useStyles = makeStyles((theme) => ({
     fontSize: ".8rem",
     paddingLeft: theme.spacing(0.5),
   },
+  fieldLabelSmall: {
+    width: "100%",
+    color: theme.palette.text.secondary,
+    fontSize: ".7rem",
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js:66]: the issue says to make smaller, the existing size is .8rem. I tried .6rem and it looked a bit too small.

**Review by mddilley — APPROVED** [review]
🚀 This one is packed with nice touches! Love seeing that title handling and the relative data looks ✨

File: moped-editor/src/utils/dateAndTime.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const MILLS_CONVERSION = {
  * Format millseconds into a relative description
  * @param {integer} millsElapsed - the elapsed time in milliseconds to format
  * @param {string} unitName - the name of the units to convert to - the string must
- * be defined in the MILLS_CONVERSION objeect
+ * be defined in the MILLS_CONVERSION object
```
Comment (johnclary) [inline moped-editor/src/utils/dateAndTime.js:79]: eeeek

File: moped-editor/src/views/projects/projectView/ProjectView.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ const useStyles = makeStyles((theme) => ({
     marginRight: theme.spacing(2),
   },
   title: {
-    height: "3rem",
+    minHeight: "3rem",
```
Comment (johnclary) [inline moped-editor/src/views/projects/projectView/ProjectView.js:81]: ✨

**Review by johnclary — APPROVED** [review]
i love it so much! these lil changes makes a huge difference!

this feels like clear scope creep, but i'll ask anyway: what do you think about adding the relative timestamps to the status modal?

Screenshot 2023-07-19 at 11 06 06 AM

**Review by frankhereford — APPROVED** [review]
This looks fantastic, thank you Chia!

The long project name reminded me of [this](https://www.youtube.com/watch?v=dCGkqUr1kbY)!

**Comment by johnclary** [comment]
@chiaberry good point. needs scoping! 🚢 🚢 🚢

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js (hunk 1 above)
Comment (johnclary) [reply moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js:66]: i do me; you do you

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js [hunk 4]
```diff
@@ -50,11 +50,8 @@ const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
           {!!statusUpdate && (
             <div>
               <div>{parse(String(statusUpdate))}</div>
-              <span className={classes.fieldAuthor}>{addedBy}</span>
-              <span className={classes.fieldLabel}>
-                {makeUSExpandedFormDateFromTimeStampTZ(
-                  dateCreated
-                ).toUpperCase()}
+              <span className={classes.fieldLabelSmall}>
+                {addedBy} - {dateCreated}
```
Comment (johnclary) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js:54]: looks great!

**Pull request merged**